plan/planbuilder.go

```diff
@@ -504,6 +506,47 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) (Plan, error) {
 	return ret, nil
 }
 
+func (b *planBuilder) buildAdminCheckTable(as *ast.AdminStmt) (*CheckTable, error) {
+	p := &CheckTable{Tables: as.Tables}
+	p.GenExprs = make(map[string]expression.Expression)
+
+	mockTablePlan := LogicalTableDual{}.init(b.ctx)
+	for _, tbl := range p.Tables {
+		tableInfo := tbl.TableInfo
+		schema := expression.TableInfo2SchemaWithDBName(b.ctx, tbl.Schema, tableInfo)
+		table, ok := b.is.TableByID(tableInfo.ID)
+		if !ok {
+			return nil, infoschema.ErrTableNotExists.GenByArgs(tbl.DBInfo.Name.O, tableInfo.Name.O)
```

Comment:
s/ tbl.DBInfo.Name.O/ tbl.DBInfo.Name.L

Reply:
I think the original is better. here is for log, not for compare.